js/views/drawerView.js

```diff
@@ -17,7 +17,8 @@ class DrawerView extends Backbone.View {
       role: 'dialog',
       'aria-modal': 'true',
       'aria-labelledby': 'drawer-heading',
-      'aria-hidden': 'true'
+      'aria-hidden': 'true',
+      'aria-expanded': 'false';
```

Comment:
Is the semicolon needed here?

Otherwise working as expected thanks. Testing on Mac Safari running Voiceover.